utils/common/src/main/java/org/apache/brooklyn/util/ssh/BashCommands.java

```diff
@@ -625,6 +625,13 @@ public static String pipeTextTo(String text, String command) {
                 +"\n"+"EOF_"+id+"\n";
     }
 
+    public static String pipeTextToFile(String text, String filepath) {
+        String id = Identifiers.makeRandomId(8);
+        return "cat > " + filepath + " << EOL_" + id + "\n"
```

Comment:
`cat > \"" + filepath + "\" ...` would be preferable to handle paths with spaces.